style.css

```diff
@@ -257,23 +285,53 @@ div.range-slider input[type=range]::-moz-focus-outer {
 	font-size: 12px;
 	color:gray;
 }
+.card-body {
+	height: 150px;
+	position:relative;
+}
 .card-img-top{
 	border-radius:10px 10px 0 0;
+	height:126px;
 }
 .card-body .price{
 	font-size:16px;
 	color:orange;
 }
+.price {
+	position:absolute;
+	bottom: 5%;
+	left:5%;
+	right:5%;
+}
+.foodCategory {
+	position:absolute;
+	font-family:Georgia, serif;
+	bottom: 20%;
+	left:10%;
+	right: 10%;
+}
 .card-text {
 	margin-bottom: 5px;
 }
 div.col-md-3 {
-	padding: 5px;
+	padding: 2px;
+	max-width: 200px;
 }
 .foodContents {
-	margin-right: 10%;
-	margin-left: 10%;
+	margin: 0 auto;
+}
+div.row {
+	width: fit-content;
+	margin: 0 auto;
+	align-content: center;
+  display: flex;   
+  flex-wrap: wrap;
 }
 button.navbar-toggler{
 	background-color: rgba(255,255,255,0.4);
+}
+#sliderValue1, #sliderValue2{
+  font-size: 10px;
+  color:gray;
+
 }
```

Comment:
I think you should refactor this file because you have to many rules with imporant. Are all those '!important' really necessary?